### PR TITLE
Small Fix: 2018 Summit link was added twice

### LINKED ISSUE #582: 2018 summit link added twice
### Expected results
All links added once
### Actual results
2018 summit link added twice
### Steps to reproduce the issue
Remove one
## Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #582

### CODE REVIEW

**Comment by saarthakchats** [comment]
@isuruAb

**Review by nikit19 — CHANGES_REQUESTED** [review]
LGTM just make sure you link this pull request to the issue it fixes

**Review by vaibhavsingh97 — COMMENTED** [review]
LGTM! But where is the issue for this PR? Please link issue with the PR

**Comment by saarthakchats** [comment]
#582

**Comment by saarthakchats** [comment]
This is the issue, Please merge

**Comment by saarthakchats** [comment]
@nikit19 @vaibhavsingh97

**Comment by gabru-md** [comment]
Why have you closed this PR?

File: index.html [hunk 1]
```diff
@@ -502,6 +502,11 @@ <h1 class="bold mb0">FOSSASIA <br>Based in Asia.</h1>
                     <i class="fa fa-vk" aria-hidden="true"></i>
                 </a>
 			</li>
+			  <li>
+			 <a href="https://in.pinterest.com/pin/390476230184177529/">
```
Comment (vaibhavsingh97) [inline index.html:506]: Why Pinterest? It's a specific post

**Comment by saarthakchats** [comment]
Made a new one #584

**Comment by saarthakchats** [comment]
@gabru-md

**Pull request closed without merging**